app/views/users/dashboard.html.erb

```diff
@@ -1,47 +1,59 @@
+<style>
+  .orcid-label {
+    color:#a4a4a4;
+    margin-top: -10px;
+  }
+
+  .dataset-section {
+    background-color: rgb(226, 223, 223);
+    margin-top: 25px;
+    margin-bottom: 25px;
+  }
+
+  .dashboard-options {
+    margin-top: -25px;
+    margin-bottom: -25px;
+  }
+
+  .dataset-table {
+    margin-left: 10px;
+  }
+</style>
+
+<% if notice.present? %>
+  <p id="notice"><%= notice %></p>
 <% end %>
 
-<p>Net ID: <%= @user.uid %></p>
-<p>Display name: <%= @user.display_name %></p>
-<p>Full name: <%= @user.full_name %></p>
-<p>Default collection: <%= @user.default_collection.title %></p>
-
-<div class="text-left">
-  <%= link_to 'Edit', edit_user_path(@user), :class => "btn btn-primary", :role => "button" %>
-</div>
-
-<% if @my_works.count > 0 %>
-  <h2>My Datasets</h2>
-  <% @my_works.each do |ds| %>
-    <p><%= link_to(ds.title, work_path(ds)) %> (<%= ds.created_by_user.uid %>), <%= ds.state %></p>
-  <% end %>
+<h2>Welcome, <%= @user.display_name %></h2>
+<% if @user.orcid.blank? %>
+  <p class="orcid-label">You do not have an  ORCID iD on file. <%= link_to("Add one", edit_user_path(@user)) %> to your profile.</p>
 <% end %>
 
-<div class="text-left">
+<div class="text-left dashboard-options">
   <%= link_to 'Add New Dataset', work_new_submission_path, class: "btn btn-primary", :role => "button" %>
 </div>
 
-<% if @awaiting_works.count > 0 %>
-<h2>Datasets waiting to be approved</h2>
-<% @awaiting_works.each do |ds| %>
-    <p><%= link_to(ds.title, work_path(ds)) %> (<%= ds.created_by_user.uid %>), <%= ds.state %></p>
-<% end %>
+<% if @my_unfinished_works.count > 0 %>
+  <h2 class="dataset-section">Unfinished Submissions</h2>
+  <%= render partial: "dataset_table", locals: {table_id: "my_unfisnihed_datasets", datasets: @my_unfinished_works} %>
 <% end %>
 
-<% if @withdrawn_works.count > 0 %>
-<h2>Datasets to have been withdrawn</h2>
-<% @withdrawn_works.each do |ds| %>
-    <p><%= link_to(ds.title, work_path(ds)) %> (<%= ds.created_by_user.uid %>), <%= ds.state %></p>
-<% end %>
+<% if @my_completed_works.count > 0 %>
+  <h2 class="dataset-section">Completed Submissions</h2>
+  <%= render partial: "dataset_table", locals: {table_id: "my_completed_datasets", datasets: @my_completed_works} %>
 <% end %>
 
-<!-- ETDS, Lib PDFs, and such could be added here as separate sections -->
-
+<% if @my_withdrawn_works.count > 0 %>
+  <h2 class="dataset-section">Withdrawn Submissions</h2>
+  <%= render partial: "dataset_table", locals: {table_id: "my_withdrawn_datasets", datasets: @my_withdrawn_works} %>
+<% end %>
 
+<script type="text/javascript">
+  $(function() {
+    // https://www.datatables.net/reference/option/
+    var dataset_options = {searching: false, paging: false, info: false};
+    $('#my_unfisnihed_datasets').DataTable(dataset_options);
+    $('#my_completed_datasets').DataTable(dataset_options);
+    $('#my_withdrawn_datasets').DataTable(dataset_options);
```

Comment:
Add some point we will want to tweak the options for these tables, but for now I am disabling them to keep the UI as clean as possible.